test/lib/util.test.js

```diff
@@ -104,6 +106,8 @@ describe('download_image', function() {
     });
 });
 
+// FIXME this text intermittently fails because it reaches out to an external
+// server. We should replace it with gzipped content served locally
```

Comment:
Fair enough - I didn't want to have copyrighted content in our repo and I didn't have a good way to simulate a gzipped server at the time which is why it's a link.